src/parser.ts

```diff
@@ -1576,6 +1582,8 @@ export class Parser extends DiagnosticEmitter {
     startPos: i32 = -1,
     signatureStart: i32 = -1
   ): FunctionExpression | null {
+    this.checkFunctionParameterDuplicates(parameters);
```

Comment:
@dcodeIO I moved check to `parseFunctionExpressionCommon` from `parseFunctionExpression`. But may be this cause to redundant checks. Need your advise